iml-services/iml-device/src/main.rs

```diff
@@ -90,27 +86,31 @@ async fn main() -> Result<(), ImlDeviceError> {
 
     let pool = iml_orm::pool().unwrap();
 
+    // You may observe spurious updates.
+    // This means that, when a single device is mounted, updated device tree may come in here several times from a single host.
+    // This is due to two reasons:
+    // 1. The swap-emitter service on the storage node fires once per minute, so there will be an update of the tree at agent side at that interval no matter what.
+    // 2. When device-scanner starts it does not fill any of the state, so there will always be data sent for the first update (state #2 below).
+    //
+    // Device scanner switches its (previous, current) states like this:
+    // #       1               2                  3
+    // state   (None, None) -> (None, initial) -> (initial, initial)
+    // A change is observed when agent is polled while it's in the middle state, and it will be in that state for a minute until swap-emitter fires.
+    //
+    // In summary, you can observe up to a minute of incoming updates, with a period of roughly 10 seconds.
+    // The period is usually a bit longer due to overhead. So a max of 5 updates has been observed in practice.
+    //
```

Comment:
I'm actually fixing this now, so we shouldn't need this comment.

Reply:
Done